docroot/sites/medicine.uiowa.edu/modules/medicine_core/medicine_core.install

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * @file
+ * Install tasks.
+ */
+
+use Drupal\redirect\Entity\Redirect;
+
+/**
+ * Medicine redirects.
+ */
+function medicine_core_update_10001() {
```

Comment:
I believe this is supposed to be a hook_install function rather than hook_update_N function. The module is being added as part of this PR. It will get turned on in production. An update hook will not run in that case, so this would be counterproductive to the whole process being the approach.